Domain/ProductManagement/Inventory.cs

```diff
@@ -0,0 +1,261 @@
+﻿using LenaInventoryManagementSystem.Domain.ProductManagement;
+using System.ComponentModel;
+using System.ComponentModel.Design;
+
+namespace LenaInventoryManagementSystem.Domain
+{
+    public class Inventory
+    {
+        private static List<Product> products = new();
```

Comment:
I see this inventory file to have many responsibilities. It doesn't the "UI" stuff, where you want to get input and print output to the user, and also you are doing the business logic for the inventory functionality. Lets separate these, think about create additional class and see how you would separate these concerns :D

Reply:
also, keep in mind, printing to screen should only be the responsibility of the class that does the input/output stuff. Other services/classes should just return enough info for the caller to handle the printing stuff.